web/portal/client/cypress/e2e/HuntGroup/HuntGroupMember/HuntGroupMember.cy.js

```diff
@@ -14,6 +14,8 @@ describe('HuntGroupMember', () => {
 
     cy.get('header').should('contain', 'Hunt Groups');
 
+    cy.usePactWait(['getHuntGroup']);
```

Comment:
this should be deleted, I'm not able to see where is declared 'getHuntGroup'

Reply:
#https://github.com/irontec/ivozprovider/blob/a8feac521dee14b1bb23ad549d93a707daa67142/web/portal/client/cypress/support/commands/prepareGenericPactInterceptors.js#L179-L181

Test would fail if it weren't declared. Isn't it?

Reply:
We can wait for this to pass using as you have written "usePactWait", but right now we instantiate "prepareGenericPactInterceptors" to intercept each of those requests, so you don't need to wait for this to happen, We actually listen in the background